### DISCUSSION TITLE
Tasmota ESP32-Cam with TM1637 LED-display

### DISCUSSION THREAD

**Discussion by ELK169** [arendst/Tasmota#22142] [Support request]
TLDR: display stays dark
I uncommented USE_DISPLAY and USE_DISPLAY_1637 in my_user_config.h. After compiling and uploading, the "Toggle 2" appears. In the next step I connected the display to 5V, GND, GPIO14 (CLK) and GPIO 15 (DIO) and assigned the two pins in the template. Then I issued the commands as listed in the tasmota documentation (displaymodel, displaywidth and so on), which were recognised (MQTT message is being generated. When I try "displayxxxxx", this will not happen).
But: the display stays dark. I connected my oscilloscope to CLK and DIO and can't see any data transfer when I issue "displaynumber 1234".
In order to localize the problem I changed GPIO to "Relay 2" and then I could switch that GPIO port through the then-new "Toggle 3" button, so there is no basic problem on the ESP32. I tried 3 different 4 digit displays. And I disabled SPI, since it seems to use these pins (I do not need the card reader). My first tries were using GPIO 12 and GPIO 13, but then the EPS32 didn't start at all.
Any ideas?

Edit: in platformio_override.ini "tasmota32" is uncommented as advised, not "tasmota32-webcam". So I uncommented USE_WEBCAM, too, and the camera works.

**Comment by Jason2866** [comment]
You can't add a display driver to the env Webcam. Read the documentation how to make a customized build.

**Comment by ELK169** [comment]
as I wrote in the last line, beginning with "Edit", I did not use the env Webcam. I used Tasmota32 and activated the Webcam by uncommenting "USE_WEBCAM". This worked without a problem.
Do you have any hints where is the right place to look in the documentation?

**Comment by Jason2866** [comment]
https://tasmota.github.io/docs/Compile-your-build/#customize-your-build
